CONTRIBUTING.md

```diff
@@ -0,0 +1,49 @@
+# Contribution guidelines
+
+Ongoing development on this project is managed by this project board: https://github.com/neontribe/www/projects/6.
+
+## Getting started on development
+
+Any issues in the `To do` column are assumed to be important enough and have been prioritised.
```

Comment:
Is going to need changing if today's meeting approves the removal of the "to do" column